app/konnect/org-management/okta-idp.md

```diff
@@ -0,0 +1,90 @@
+
+### Prepare the Okta application
+
+Create a new application in Okta to manage Konnect Cloud account integration.
+
+1. Sign in to your [Okta developer account](https://developer.okta.com/).
+1. From the left menu, select **Applications**, then **Create Integration**.
+1. Select the application type:
+    1. Under **Sign-on method**, select **OIDC - OpenID Connect**.
+    1. Under **Application Type**, select **Web Application**.
+1. Click **Next**. Configure the application:
+    1. Create a unique name for your application.
+    1. Under **Grant Type**, select **Authorization Code**.
+    1. In both the **Sign-in redirect URIs** and
+**Sign-out redirect URIs** fields, enter:
+
+      `https://<div contenteditable="true">{YOUR_KONNECT_APP_DOMAIN}</div>/login`.
+
+      Replace `{YOUR_KONNECT_APP_DOMAIN}` with your own domain.
+1. Save your settings to generate connection details.
+
+    Leave this page open. You'll need the details here to configure your Konnect
+    Cloud account.
+
+### Set the default IdP for Konnect Cloud
+
+1. In a separate browser tab, log in to [Konnect Cloud](https://konnect.konghq.com).
+1. Open ![settings icon](/assets/images/icons/konnect/konnect-settings.svg){:.inline .no-image-expand}
+**Settings**, then **Identity Management**.
+1. Select **Okta**.
+1. Copy and paste the **Client ID** and **Client Secret** from your Okta
+application into Konnect Cloud.
+
+    See the [Okta developer documentation](https://developer.okta.com/docs/guides/find-your-app-credentials/findcreds/)
+    to learn more about client credentials in Okta.
+
+1. Create the **Issuer URL**:
+    1. Copy the **Okta domain** from your Okta application.
+    1. Paste it into the **Issuer** field in Konnect using the following format:
+
+        `https://<div contenteditable="true">{OKTA_DOMAIN}</div>/oauth2/default`
+1. Save your changes.
+1. Copy the generated login URL.
+
+    This is your organization's new login URL. Provide this URL to users to let them
+    log in to Konnect with Okta.
+
+1. Paste the URL into a browser window.
+
+    This should load an Okta application login screen.
+
+1. Log in with your Okta credentials.
+
+### Enable Konnect Cloud as a dashboard app in Okta
+
+If you want your users to have easy access to Konnect Cloud alongside their other apps,
+you can add it to your Okta dashboard.
+
+1. Sign in to your [Okta developer account](https://developer.okta.com/).
+1. Click **Applications**, then open your Konnect Cloud Okta application.
+1. Scroll to **General Settings** and click **Edit** to configure the following:
+    1. **Grant Type**: Select the **Implicit (Hybrid)** checkbox.
+    1. **Login Initiated by**: Select **Either Okta or App** from the dropdown.
+    1. **Aplication visibility**: Select **Display application icon to users**.
+    1. **Initiate login URI**: Paste your organization's login URL.
+        You can find this URL in Konnect Cloud under **Settings** >
+        **Identity Management**.
```

Comment:
I think we need to add a section to this topic to describe how to setup the proper OIDC claims to be sent from Okta to Konnect. They are outlined in this doc https://docs.google.com/document/d/1fsF9vzmEfUKt4yHdsWQhOrw4QDJnPouV8kSzhJTMoqg/edit# and I tagged Lena in the appropriate section. The KX team can help walk through these steps if needed.